examples/epstein_civil_violence/epstein_civil_violence/agent.py

```diff
@@ -78,14 +78,9 @@ def step(self):
         self.update_neighbors()
         self.update_estimated_arrest_probability()
         net_risk = self.risk_aversion * self.arrest_probability
-        if (
-            self.condition == "Quiescent"
-            and (self.grievance - net_risk) > self.threshold
-        ):
+        if self.grievance - net_risk > self.threshold:
```

Comment:
Is this to simplify the code because regardless of the prior condition, the final condition is going to be "Active" anyway?

Reply:
Aye, the condition is going to be one or the other as an agent choice, and it's entirely dependent on the parameter net_risk on any given step as the others are static, so it makes sense to make it if/else with quiescent as baseline.